src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+  }
+
+  &__star:last-child {
+    margin-bottom: 0;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
Ensure that the image paths are correct and relative to the SCSS file location. Verify that the images exist at the specified paths.